opentelemetry-otlp/src/lib.rs

```diff
@@ -214,12 +214,15 @@ pub use crate::exporter::{
 use opentelemetry::sdk::export::ExportError;
 
 #[cfg(feature = "grpc-sys")]
-pub use crate::exporter::grpcio::GrpcioExporterBuilder;
+pub use crate::exporter::grpcio::{Compression, Credentials, GrpcioExporterBuilder};
```

Comment:
There are a few things to say here:

1. They should probably be prefixed with `Grpcio` as they are specific to that exporter
2. They were public but not accessible. There was no way to call `with_compression` or `with_credentials` because the types couldn't be instantiated. In the doc you can see it because the types are not clickable: 
![image](https://user-images.githubusercontent.com/3018448/131698689-03ba873c-8641-4b29-8dfd-d02c0e112969.png)